packages/core/src/database/postgres/service.test.ts

```diff
@@ -427,6 +438,46 @@ describe.skipIf(shouldSkip)("postgres database", () => {
     await database.kill();
   });
 
+  test("default values are populated during insertion", async (context) => {
```

Comment:
Nice test - is there a way to tell if a column has a default value, similar to `getTableNames` function used in this test file

Reply:
hm. i don't know. are you asking for type generation purposes? i would presume that it would show up in a `\d table_name` command on postgres

Reply:
i was able to creae one [here](https://github.com/ponder-sh/ponder/pull/1085/files#diff-ae6cfa84163e684033b864abfb27a9245eb057f932f6e2b8b69135bfb5b05fc6R1026)